bundles/org.openhab.binding.kostalinverter/src/main/resources/OH-INF/config/SecondGeneration.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+	<config-description uri="thing-type:kostalpiko1020_config">
+		<parameter name="url" type="text" required="true">
+			<label> Inverter URL</label>
+			<description>URL to inverter.</description>
+			<context>url</context>
+		</parameter>
+		<parameter name="username" type="text" required="false">
+			<label>Username</label>
+			<description>The username for the inverter.</description>
+		</parameter>
+		<parameter name="password" type="text" required="false">
+			<label>Password</label>
+			<description>The password for the inverter.</description>
+			<context>password</context>
+		</parameter>
+		<parameter name="refreshInterval" type="integer" min="1" unit="s" required="false">
```

Comment:
This should be `required="true"`, as if the user doesn't specify it in textual configuration, it will be 0.